co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/V2xRegistrationMessage.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2023 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+ 
+package org.eclipse.mosaic.fed.infrastructure.ambassador;
+
+/**
+ * A message to be sent by Infrastructure Device when it registers with the carma-mosaic ambassador
+ * NOTE: TODO See carma.ambassador for reference
+ */
+public class V2xRegistrationMessage {
+    private String rxMessageIpAddress;
+    private int rxMessagePort;
```

Comment:
This message also requires a location and a time port to send time sync messages to.

Reply:
added